### PR TITLE
[docs] Add ExecRaw annotation documentation.

### PR DESCRIPTION
**Pull request by ultimike**
The title says it all....

<a href="https://gitpod.io/#https://github.com/ddev/ddev/pull/4735"></a>

### CODE REVIEW

File: docs/content/users/extend/custom-commands.md [hunk 1]
```diff
@@ -218,6 +218,14 @@ If your container command should run from the directory you are running the comm
 
 Example: `## HostWorkingDir: true`
 
+### “ExecRaw” Annotation (Container Commands Only)
+
+Use `ExecRaw: true` to pass arguments for the command you want to run directly to the command in the container as-is. 
+
+For example, when ExecRaw is true, `ddev yarn --help` returns the help for `yarn`, not DDEV's help for the `ddev yarn` command.
```
Comment (rfay) [inline docs/content/users/extend/custom-commands.md:225]:
```suggestion
For example, when ExecRaw is true, `ddev yarn --help` returns the help for `yarn`, not DDEV's help for the `ddev yarn` command.

Most commands will work better with this annotation. This is recommended for almost all container commands.
```

**Review by rfay — APPROVED** [review]
Just added one suggestion you can commit if you like it.

File: docs/content/users/extend/custom-commands.md [hunk 2]
```diff
@@ -218,6 +218,14 @@ If your container command should run from the directory you are running the comm
 
 Example: `## HostWorkingDir: true`
 
+### “ExecRaw” Annotation (Container Commands Only)
+
+Use `ExecRaw: true` to pass arguments for the command you want to run directly to the command in the container as-is.
```
Comment (mattstein) [inline docs/content/users/extend/custom-commands.md:223]: This sentence seems a little tricky to parse. Maybe this whittles it down without losing meaning?

```suggestion
Use `ExecRaw: true` to pass command arguments directly to the container as-is. 
```

File: docs/content/users/extend/custom-commands.md (hunk 1 above)
Comment (mattstein) [reply docs/content/users/extend/custom-commands.md:225]: > Most commands will work better with this annotation. This is recommended for almost all container commands.

Two things:

1. This reads like “you should mostly use this” without qualifying why. Is it possible to give an example of when this is and is not ideal to help someone navigate their own decision?
2. Ideally recommendations can be [active third person](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#use-active-third-person); “this is recommended” → “we recommend”.

**Review by mattstein — CHANGES_REQUESTED** [review]
Added my nitpickery!

File: docs/content/users/extend/custom-commands.md (hunk 1 above)
Comment (rfay) [reply docs/content/users/extend/custom-commands.md:225]: Thanks @mattstein - I updated. But shouldn't that be called "active first person"?

**Pull request merged**